src/libraries/System.Management/tests/System/Management/ManagementObjectTests.cs

```diff
@@ -65,6 +65,10 @@ public void Set_Property_Win32_ComputerSystem()
         [OuterLoop]
         public void Invoke_Instance_And_Static_Method_Win32_Process()
         {
+            if (PlatformDetection.IsWindows10Version22000OrGreater) {
+                // https://github.com/dotnet/runtime/issues/70414
+                throw new SkipTestException("Unstable on Windows 11");
+            }
```

Comment:
There are a couple of these in System.Diagnostics.EventLog (https://github.com/dotnet/runtime/issues/58829). Our assumption is an OS bug as our code didn't change. 

I think disabling for Windows 11 makes sense in this situation. However we should probably leave the issue open (remove "Fixes https://github.com/dotnet/runtime/issues/70414" from description above). Ideally we'd also disable in such a way that it re-enables itself on Windows "12" but  what would that be -- 23000?

Reply:
Ideally we'd also open bugs against the OS, but for flaky issues like this that's pretty expensive and given it's a legacy API it might not be a high priority.

Reply:
> Ideally we'd also disable in such a way that it re-enables itself on Windows "12" but what would that be -- 23000?

I noticed the dev channel releases use 25000+ in their build numbers. Is it safe to assume 23000 is reserved for Windows 12?